br/pkg/backup/client.go

```diff
@@ -410,7 +410,16 @@ func (bc *Client) BuildBackupRangeAndSchema(
 	isFullBackup bool,
 ) ([]rtree.Range, *Schemas, []*backuppb.PlacementPolicy, error) {
 	if bc.checkpointMeta == nil {
-		return BuildBackupRangeAndSchema(storage, tableFilter, backupTS, isFullBackup, true)
+		ranges, schemas, policies, err := BuildBackupRangeAndSchema(storage, tableFilter, backupTS, isFullBackup, true)
+		if err != nil {
+			return nil, nil, nil, errors.Trace(err)
+		}
+		// Add keyspace prefix to BackupRequest
+		for i := range ranges {
```

Comment:
Is it possible to move prefixing keyspace logic into `BuildBackupRangeAndSchema`?

Reply:
done